cloudinit/config/cloud-init-schema.json

```diff
@@ -1343,6 +1343,303 @@
+              "additionalItems": false,
+              "minItems": 1,
+              "minProperties": 1,
+              "additionalProperties": {
+                "oneOf": [
+                  {"type": "string"},
+                  {"type": "array", "items": {"type": "string"}}
+                ]
+              }
+            },
+            "squashfuse_in_container": {
+              "type": "boolean",
+              "description": "**Development only**: The ``squashfuse_in_container`` boolean can be set true to install squashfuse package when in a container to enable snap installs. Default: ``false``."
+            }
+          }
+        }
+      }
+    },
+    "cc_spacewalk": {
+      "type": "object",
+      "properties": {
+        "spacewalk": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "server": {
+              "type": "string",
+              "description": "The Spacewalk server to use"
+            },
+            "proxy": {
+              "type": "string",
+              "description": "The proxy to use when connecting to Spacewalk"
+            },
+            "activation_key": {
+              "type": "string",
+              "description": "The activation key to use when registering with Spacewalk"
+            }
+          }
+        }
+      }
+    },
+    "cc_ssh_authkey_fingerprints": {
+      "type": "object",
+      "properties": {
+        "no_ssh_fingerprints": {
+          "type": "boolean",
+          "default": false,
+          "description": "If true, SSH fingerprints will not be written. Default: ``false``"
+        },
+        "authkey_hash": {
+          "type": "string",
+          "default": "sha256",
+          "description": "The hash type to use when generating SSH fingerprints. Default: ``sha256``"
+        }
+      }
+    },
+    "cc_ssh_import_id": {
+      "type": "object",
+      "properties": {
+        "ssh_import_id": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "The SSH public key to import"
+          }
+        }
+      }
+    },
+    "cc_ssh": {
+      "type": "object",
+      "properties": {
+        "ssh_keys": {
+          "type": "object",
+          "description": "A dictionary entries for the public and private host keys of each desired key type. Entries in the ``ssh_keys`` config dict should have keys in the format ``<key type>_private``, ``<key type>_public``, and, optionally, ``<key type>_certificate``, e.g. ``rsa_private: <key>``, ``rsa_public: <key>``, and ``rsa_certificate: <key>``. Not all key types have to be specified, ones left unspecified will not be used. If this config option is used, then separate keys will not be automatically generated. In order to specify multiline private host keys and certificates, use yaml multiline syntax.",
+          "patternProperties": {
+            "^(rsa|dsa|ecdsa|ed25519)_(public|private|certificate)$": {
```

Comment:
This regex looks a lot more strict that what our cc_ssh module documents as supported. I'll have to reread this content again to better understand whether this incorrectly excludes config directives.